generate.go

```diff
@@ -0,0 +1,55 @@
+package main
```

Comment:
this should not be in the top level of the repo

Reply:
Will update this